src/Tizen.NUI/src/internal/Application/Application.cs

```diff
@@ -1090,6 +1090,19 @@ public static Application NewApplication(string[] args, string stylesheet, NUIAp
             return instance;
         }
 
+        public static Application NewApplication(string stylesheet, NUIApplication.WindowMode windowMode, Rectangle positionSize, WindowType type)
+        {
+            if (instance)
+            {
+                return instance;
+            }
+            Application ret = New(1, stylesheet, windowMode, positionSize, type);
```

Comment:
could you add the first parameter(`1`) meaning with comment? or can be defined as constant like `private const int SomeMeaningfulName = 1;`

Reply:
It just is dummy value. I referred the other constructor in this file.